### ISSUE TITLE
SSH Key copied in container got 0777 perms and is ignored

### ISSUE THREAD

**Issue by Oza94** [dockito/vault#14]
Hello,

I try to setup vault for my project but I think I'm missing something. I'm on Windows and have a fresh install of docker toolbox (docker 1.9), I'm running dockito/vault with

```
docker run -p 172.17.0.1:14242:3000 -v ~/.ssh:/vault/.ssh dockito/vault
```

Then I have this Dockerfile which tries to clone a private repo

```
# because it is a Dockerfile for my app
FROM node:0.12

# installs Dockito Vault ONVAULT utility
# https://github.com/dockito/vault
RUN apt-get update -y && \
    apt-get install -y curl && \
    curl -L https://raw.githubusercontent.com/dockito/vault/master/ONVAULT > /usr/local/bin/ONVAULT && \
    chmod +x /usr/local/bin/ONVAULT

ENV VAULT_PORT tcp://172.17.0.1:14242
ENV VAULT_SSH_KEY vault_rsa

RUN ONVAULT git clone git@bitbucket.org:myorg/myprivaterepo.git

CMD ["ls"]
```

But the clone fails in the container

![build screenshot](http://i.imgur.com/XYKsGrl.png)

However, a `ssh -T -i ~/.ssh/vault_rsa git@bitbucket.org` from the docker deamon VM is fine and `git clone repo` works too. What am I doing wrong? Sorry if this is obvious I'm not very comfortable with Docker yet

**Comment by Oza94** [comment]
Ok sorry I missed the "This private key will be ignored" I feel dumb and tired now --

**Comment by Oza94** [comment]
Ok so adding a `chmod 600 ~/.ssh/*` solve my issue, see my custom ONVAULT script [here](https://gist.github.com/Oza94/bbca89f02ec568dd10ac#file-onvault-1-9-L36). I'm not sure I should fix something on my side.

@pirelenito would you accept a PR on that?

**Comment by maxclaus** [comment]
@Oza94 Usually the "recommended" permission to ssh files is 600. So you "should" already being using this in your original files. Then this problem would no happen (at least I already use the permissions this way and have never seen this problem with dockito-vault). But since this has happened to you. I guess could happen to someone else. So your pull request could be helpful.

**Comment by csxphil** [comment]
So, I might be inexperienced with setting permissions, but I had to use @Oza94 one line fix in a fork to get this to work when building a dockerfile on a windows 7 machine. If this project ever implements this, I would drop the fork and move back to the original in a heartbeat.

**Comment by maxclaus** [comment]
Implemented in version 1.7.2.

**Comment by csxphil** [comment]
Thank you, my issue is resolved!
